test/tasks.spec.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env mocha */
+const expect = require('chai').expect
+const lib = require('../lib/helper-functions')
+
+describe('tasks', () => {
+  describe('when running copy-to-destination', () => {
+    // Check to have all the components in dist and nothing else
+    it('should copy all components to dist', () => {
+      expect(lib.DistComponentList).to.deep.equal(lib.SrcComponentList)
+    })
+
+    // Check to have all the components (plus all, globals, icons) in packages and nothing else
+    it('should copy all components plus "all", "globals" and "icons" to packages', () => {
+      lib.SrcComponentList.push('all', 'globals', 'icons')
+      expect(lib.PackagesComponentList).to.have.same.members(lib.SrcComponentList)
+    })
+  })
+
+  function _Published (componentName) {
+    // Check component files in dist
+    describe('when publishing ' + componentName, () => {
+      it('should copy the right files to dist', () => {
+        let c = lib.compareComponentDist(componentName)
+        expect(c.distComponentFiles).to.have.same.members(c.srcComponentFiles)
+      })
+
+      // Check component files in packages
+      it('should copy the right files to packages', () => {
+        let c = lib.compareComponentPackage(componentName)
+        expect(c.packagesComponentFiles).to.have.same.members(c.srcComponentFiles)
+      })
+    })
+  }
+
+  for (let i = 0; i < lib.SrcComponentList.length; i++) {
```

Comment:
Could this be done with a `forEach`?

Reply:
`for` is faster, `forEach` easier to read. If we're fine with `forEach` and use it consistently I can comply.

Reply:
I don't have strong opinions, I just worry that this bit of the test is hard to read at the minute. I'd also be tempted to rename `_Published` to make it clearer what's going on – e.g. `defineTestsForComponent` or something.